src/main/java/ch/njol/skript/expressions/ExprCreeperMaxFuseTicks.java

```diff
@@ -1,102 +0,0 @@
-/**
- *   This file is part of Skript.
- *
- *  Skript is free software: you can redistribute it and/or modify
- *  it under the terms of the GNU General Public License as published by
- *  the Free Software Foundation, either version 3 of the License, or
- *  (at your option) any later version.
- *
- *  Skript is distributed in the hope that it will be useful,
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *  GNU General Public License for more details.
- *
- *  You should have received a copy of the GNU General Public License
- *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
- *
- * Copyright Peter Güttinger, SkriptLang team and contributors
- */
-package ch.njol.skript.expressions;
-
-import org.bukkit.entity.Creeper;
-import org.bukkit.entity.LivingEntity;
-import org.bukkit.event.Event;
-import org.eclipse.jdt.annotation.Nullable;
-
-import ch.njol.skript.Skript;
-import ch.njol.skript.classes.Changer.ChangeMode;
-import ch.njol.skript.doc.Description;
-import ch.njol.skript.doc.Examples;
-import ch.njol.skript.doc.Name;
-import ch.njol.skript.doc.Since;
-import ch.njol.skript.expressions.base.SimplePropertyExpression;
-import ch.njol.util.coll.CollectionUtils;
-
-@Name("Creeper Max Fuse Ticks")
-@Description("The max fuse ticks that a creeper has.")
-@Examples("set target entity's max fuse ticks to 20 #1 second")
-@Since("2.5")
-public class ExprCreeperMaxFuseTicks extends SimplePropertyExpression<LivingEntity, Long> {
-	
-	static {
-		if(Skript.methodExists(LivingEntity.class, "getMaxFuseTicks"))
```

Comment:
This syntax has never actually worked since LivingEntity doesn't have `getMaxFuseTicks`

Reply:
![image](https://github.com/SkriptLang/Skript/assets/25848425/b2759996-e9a1-45bf-abf2-83fb508ed0df)

Reply:
lol